README.md

```diff
@@ -1,7 +1,7 @@
 [![Build Status](https://travis-ci.org/rallyhealth/play-test-ops.svg?branch=master)](https://travis-ci.org/rallyhealth/play-test-ops)
 [![codecov](https://codecov.io/gh/rallyhealth/play-test-ops/branch/master/graph/badge.svg)](https://codecov.io/gh/rallyhealth/play-test-ops)
 
-| play23-test-ops-core | play25-test-ops-core | play25-test-ops-core |
+| play23-test-ops-core | play25-test-ops-core | play27-test-ops-core |
```

Comment:
I also think that play23 is play25 and play25 is play26.

Reply:
Fixed on master.